AGENT_CHANGELOG.md

```diff
@@ -440,7 +478,7 @@
 * Amazon Kafka [3.3.0](https://github.com/DataDog/integrations-core/blob/master/amazon_msk/CHANGELOG.md)
 * Apache [4.2.1](https://github.com/DataDog/integrations-core/blob/master/apache/CHANGELOG.md)
 * Avi Vantage [3.2.1](https://github.com/DataDog/integrations-core/blob/master/avi_vantage/CHANGELOG.md)
-* Datadog Checks Base [31.0.1](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/CHANGELOG.md) **BREAKING CHANGE**
```

Comment:
Also not sure why this got updated, it's for an old release.

Reply:
**The update is correct!**

I checked out the 7.45.0 tag and the updated version is the one that's in the requirements:
https://github.com/DataDog/integrations-core/blob/bb8520bc2a877a2d1fbe640bd0526378ec20e8bc/requirements-agent-release.txt#L22

@FlorentClarret you think we should file a card to investigate why this is only getting updated now?

Reply:
I just think we did not run this command when 7.49.1 came out so no, I don't think we should

Reply:
I find it unlikely that we didn't run the command since 7.45.0 (which is the release affected by the change), but investigating this may not be worth our time until we have some better example to play with.

Reply:
Oh sorry 🤦 I read 7.49.1 for some reason. Yes that's suspicious